content/docs/references/cli/_index.md

```diff
@@ -782,3 +783,29 @@ Encryption rekey status with name of the rekey object
 # dellctl encryption rekey-status myrekey
  INFO Status of rekey request myrekey = completed
 ```
+### dellctl volume list
+
+Lists local volumes metadata for a given tenant. 
```

Comment:
Please indicate that this command is currently only supported for Powerflex volumes.

Reply:
Thanks @sharmilarama, this has been updated